app/src/main/java/org/gdg/frisbee/android/chapter/MainActivity.java

```diff
@@ -444,6 +428,11 @@ void launchAppInviteActivity(Intent intent) {
         startActivity(newIntent);
     }
 
+    @Override
+    public void onChapterSelected(Chapter selectedChapter) {
+        updateSelectionFor(selectedChapter.getGplusId());
```

Comment:
here, you convert a chapter into an id and then back into a chapter. Should be made more performant.